src/codeGenerator/store.ts

```diff
@@ -19,19 +29,23 @@ export class FunctionValue implements Serializable, Writeable {
   }
 
   static onTick(name: string, commands: string[]) {
```

Comment:
If we're rewriting tick handling to use an advancement, I don't think we'll need this to be a separate static constructor anymore?

Reply:
ack